pytket/extensions/quantinuum/backends/api_wrappers.py

```diff
@@ -18,7 +18,7 @@
 
 import time
 from http import HTTPStatus
-from typing import Optional, Dict, Tuple
+from typing import Optional, Dict, Tuple, List
```

Comment:
Why do we need this?

Reply:
This was legacy change. I initially, added the function that uses L4 API in QuantinuumAPI